cwms-data-api/src/test/java/cwms/cda/data/dao/basin/BasinDaoIT.java

```diff
@@ -172,7 +172,7 @@ private static void createTestLocation(String locationName, String kind) throws
         db.connection(c -> {
             try {
                 LocationsDaoImpl locationsDao = new LocationsDaoImpl(getDslContext(c, OFFICE_ID));
-                locationsDao.storeLocation(location);
+                locationsDao.storeLocation(location, false);
```

Comment:
In the future consider providing a override that uses the default... though certainly debatable whether or not that's a good idea in the context of the our particular usage.